### ISSUE TITLE
[Feature]: Add any Paxel (All in One Tool) Mod

### ISSUE THREAD

**Issue by AbhijeetMankani** [ThePansmith/Monifactory#1282]
### Suggestion

Pretty much self-explanatory. Switching between tools is just a hassle, which can be done without. It's just a QOL feature.

**Comment by wedgevic** [comment]
The multi tool is available in IV which is basically that. It would be nice to have a quest talking about it though

**Comment by AbhijeetMankani** [comment]
> The Omni-breaker is available in IV which is basically that. It would be nice to have a quest talking about it though

Yeah, I think it was added in a recent update. I guess that'll close this issue, then.
